argus/scenarios/cloud.py

```diff
@@ -233,3 +234,23 @@ class MaasWindowsScenario(BaseServiceMockMixin, BaseWindowsScenario):
               host="0.0.0.0",
               port=2002),
     ]
+
+
+class UpdatePasswordScenario(CloudstackWindowsScenario):
+
+    """Scenario for testing the password update feature."""
+
+    @contextlib.contextmanager

```

Comment:
This should be in the base class of mocked services scenarios, in argus.scenarios.cloud.BaseServiceMixin.